### ISSUE TITLE
Feature Request: Option to use hourly forecast instead of whole day

### ISSUE THREAD

**Issue by jschlackman** [imbrianj/ready_for_rain#5]
Hello,

Would you consider adding an option to use the hourly forecast to check for rain, instead of the whole-day forecast? The use case is that we'd like to use the app to warn us of approaching rain when we open the front door to avoid being unexpectedly drenched while walking the dog. Right now we get warned if there's rain forecast at any time that day, when we only need to know if rain is coming soon.

I think the data should be available by passing "hourly" in the getWeatherFeature call instead of "forecast" and taking the first set of data to be returned, but I don't think I'm familiar enough with Groovy to try and implement this myself.

**Comment by jschlackman** [comment]
Well, I learned enough Groovy to make this happen. I also added some missing rain keywords from the WU [Forecast Description Phrases](https://www.wunderground.com/weather/api/d/docs?d=resources/phrase-glossary#forecast_description_phrases), and threw in an option for TTS notifications, just because I like having them.

These changes have all worked successfully in production in the last few weeks, so I've created pull request #6 if you'd like to incorporate the changes.
